tests/test_consume_whitespace.py

```diff
@@ -0,0 +1,39 @@
+"""All Unit tests which focus on how whitespace is handled by the calculator.
```

Comment:
[The Google style does](https://google.github.io/styleguide/pyguide.html#Comments)

"""string until line break
"""

In this case moving up the triple quotes puts it over 80 characters.